app/src/main/res/layout/user_row.xml

```diff
@@ -15,22 +16,19 @@
 
         <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="match_parent"
-            android:layout_height="match_parent">
+            android:layout_height="match_parent"
+            android:layout_gravity="center">
 
             <TextView
```

Comment:
Could you use some sort of android:gravity="center_vertical" ? this way the members' emails would start from the left. This might be just a stylistic preference though

Reply:
Done

Reply:
Thanks Luca!